main/loudmouth/APKBUILD

```diff
@@ -12,63 +12,36 @@ depends_dev="pkgconfig gnutls-dev libidn-dev libasyncns-dev check-dev autoconf"
 makedepends="$depends_dev glib-dev"
 install=""
 subpackages="$pkgname-dev $pkgname-doc"
-source="http://ftp.gnome.org/pub/gnome/sources/${pkgname}/1.4/${pkgname}-${pkgver}.tar.bz2
-	01-fix-sasl-md5-digest-uri.patch
-	03-drop-stanzas-on-fail.patch
-	04-use-pkg-config-for-gnutls.patch
-	loudmouth-1.4.3-glib-compile-fix.patch
-	gnutls-3.4.0.patch
-	"
-_builddir="$srcdir"/$pkgname-$pkgver
+source="https://mcabber.com/files/${pkgname}/${pkgname}-${pkgver}.tar.bz2"
+builddir="$srcdir"/$pkgname-$pkgver
 
 prepare() {
-	local i
-	cd "$_builddir"
-	update_config_sub || return 1
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
+	default_prepare || return
 }
 
 build() {
-	cd "$_builddir"
+	cd "$builddir"
 	autoconf
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--disable-static \
 		--with-asyncns=yes \
 		--prefix=/usr \
-		--with-libgnutls-prefix=/usr \
 		--infodir=/usr/share/info \
 		--localstatedir=/var \
 		|| return 1
```

Comment:
Could you remove the `|| return 1` statements here and elsewhere? They are not needed anymore.